lib/graphql_connector/service_classable/query.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module GraphqlConnector
+  module ServiceClassable
+    class InvalidTypeMappingError < StandardError
```

Comment:
I prefer the following syntax:
```ruby
class InvalidTypeMappingError < StandardError; end
```

Reply:
Agree that looks much nicer. Refactored it accordingly